### PR TITLE
Parameterize suite name for downstream providers that run different s…

### PR DESCRIPTION
**Pull request by jayunit100**
We typcially run a few different suites.  It would be nice if we could parameterize the suite name when we run it from a wrapper.

### CODE REVIEW

**Comment by jayunit100** [comment]
cc @kubernetes/sig-testing

**Comment by jayunit100** [comment]
xref to <- https://github.com/openshift/origin/pull/7474/files

**Comment by timothysc** [comment]
@karlkfi, @jdef this would help the extensiblity argument now.

File: test/e2e/e2e.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 // If a "report directory" is specified, one or more JUnit test reports will be
 // generated in this directory, and cluster logs will also be saved.
 // This function is called on each Ginkgo node in parallel mode.
-func RunE2ETests(t *testing.T) {
+func RunE2ETests(t *testing.T, suite string) {
```
Comment (timothysc) [inline test/e2e/e2e.go]: I need to give this a bit more thought, all this is doing is shuffling the title string around, it does really create more extensible system.  b/c if folks extend from this point it simply creates a monolith by a different name.  

Ideally I would like suite segregation so you could select by suites in a hierarchical fashion.  Other test runners have this capability, but right now it just appears that we're calling a monolith by a different name.

Comment (jayunit100) [inline test/e2e/e2e.go]: agree the string is being passed at a time which is after any real customizations are added (compilation), and before the time that customizations are removed (ginkgo focus).... :)

Really it should be an arg sent along side other user provided arguments and if not provided, it should be "kubernetes test suite".

Sounds better?

Comment (timothysc) [inline test/e2e/e2e.go]: Here is the xref for context: https://github.com/onsi/ginkgo/issues/130

Comment (timothysc) [inline test/e2e/e2e.go]: @ihmccreery Do you have thoughts around e2e-extensiblity for downstream?

Comment (ihmccreery) [inline test/e2e/e2e.go]: cc @kubernetes/goog-testing @kubernetes/sig-testing

**Comment by jayunit100** [comment]
closed now because we can use `--ginkgo.focus=[k8s.io]`

**Pull request closed without merging**